.github/workflows/versions-propagation-auto-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Versions propagation auto-merge
+on:
+  workflow_call:
+
+jobs:
+  enable-auto-merge:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'alfresco-build' && contains( github.event.pull_request.labels.*.name, 'updatebot') }}
```

Comment:
I don't think it's too opinionated: only these PRs should be auto-merged this way. Maybe this can be revisited when there are other use cases that require auto merge but I think it's good like this for now :+1: 
Actually I would even be in favor of adding a specific label on version propagation PRs because the "updatebot" label is quite generic here, but I'm not aware of when the "updatebot" label is set, maybe that matches this only use case (?)

Reply:
Yes, the `updatebot` label, is added by the tool `updatebot`  (the one we are using to propagate versions). So it should be all fine, on that end.  The CI is also adding it to PRs that are created using GH client directly.